src/Symfony/Component/Form/FormRegistry.php

```diff
@@ -82,11 +82,14 @@ public function getType($name)
 
             if (!$type) {
                 // Support fully-qualified class names
-                if (class_exists($name) && in_array('Symfony\Component\Form\FormTypeInterface', class_implements($name))) {
-                    $type = new $name();
-                } else {
-                    throw new InvalidArgumentException(sprintf('Could not load type "%s"', $name));
+                if (!class_exists($name)) {
+                  throw new InvalidArgumentException(sprintf('Could not load type "%s": class does not exist', $name));
                 }
+                else if (!in_array('Symfony\Component\Form\FormTypeInterface', class_implements($name))) {
```

Comment:
`elseif` and please move it to the previous line

Reply:
Actually, there is no need for an `elseif` as the previous condition exits anyway. So please just make this a simple `if`.